easybuild/easyconfigs/g/GROMACS/GROMACS-2020.3-CrayGNU-20.08-cuda-pat7.1.1.eb

```diff
@@ -0,0 +1,48 @@
+# contributed by Luca Marsella, Victor Holanda Rusu (CSCS)
+#
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+easyblock = 'CMakeMake'
+name = 'GROMACS'
+version = "2020.3"
+local_patversion = '7.1.1'
+versionsuffix = '-cuda-pat%s' % (local_patversion)
+
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'CrayGNU', 'version': '20.08'}
+toolchainopts = {'usempi': True, 'openmp': True, 'pic': True, 'verbose': False, 'opt': True, 'dynamic': False}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+
+# CMake dependency with dummy toolchain
+builddependencies = [
+    ('CMake', '3.14.1', '', True),
+    ('perftools-lite/%s' % local_patversion, EXTERNAL_MODULE),
+]
+
+dependencies = [
+    ('craype-accel-nvidia60', EXTERNAL_MODULE),
+    ('cray-libsci', EXTERNAL_MODULE)
```

Comment:
```suggestion
    ('craype-accel-nvidia60', EXTERNAL_MODULE),
```

Reply:
The module `cray-libsci` is loaded by default with `PrgEnv-gnu`, loaded by `CrayGNU`.